docs/notebooks/thomson.ipynb

```diff
@@ -90,6 +88,7 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
+    "incorrectly_encoded_metadata": "nbsphinx-thumbnail={\"tooltip\": \"Thomson Scattering\"}",
```

Comment:
What happened here? @StanczakDominik Do you know what this is?

Reply:
Oh bloody hell.

What happened here is that I tried to be fancy and edit this really quickly with https://github.com/goerz/jupytext.vim. I guess it handle didn't that particular metadata key for some reason.

I'm thinking it should still work, though - it added a key, but didn't remove anything. I'll remove this new addition, though. Thanks for catching it!